components/brave_new_tab_ui/components/default/braveToday/customize/SourcesListEntry.tsx

```diff
@@ -59,26 +69,32 @@ function FavIcon (props: { src?: string }) {
 
 export function FeedListEntry (props: Props) {
   const publisher = usePublisher(props.publisherId)
-  const { subscribed, setSubscribed } = usePublisherSubscribed(props.publisherId)
+  const { followed, setFollowed } = usePublisherFollowed(props.publisherId)
 
-  return <Container direction="row" justify="space-between" align='center' onClick={() => setSubscribed(!subscribed)}>
+  return <Container direction="row" justify="space-between" align='center'>
     <Flex align='center' gap={8}>
       <FavIcon src={publisher.faviconUrl?.url} />
       <Text>{publisher.publisherName}</Text>
     </Flex>
-    <ToggleButton>
-      {subscribed ? Heart : HeartOutline}
+    <ToggleButton onClick={() => setFollowed(!followed)}>
```

Comment:
Might be worth adding some padding/more gap to this button. It looks a bit odd with long names:
![image](https://user-images.githubusercontent.com/7678024/196323803-d79041e8-0379-47ef-9eae-6ce64ebe8f6a.png)

It might be because the `Flex` is collapsing the gap as things get too bit.

Reply:
We can tidy this up when we add channel images, Leo's button and fallback images